RULES.md

```diff
@@ -256,6 +256,29 @@ Any other combination raise this error.
 #### References:
 * [stops.txt specification](http://gtfs.org/reference/static/#stopstxt)
 
+<a name="OverlappingTripFrequenciesNotice"/>
+
+### OverlappingTripFrequenciesNotice
+
+Trip frequencies must not overlap in time
+
+#### References:
+
+* [GTFS frequencies.txt specification](http://gtfs.org/reference/static/#frequenciestxt)
+
+Value of field `location_type` of parent found in field `parent_station` is invalid.
```

Comment:
I think this is missing an error header, or this text shouldn't be here?

It's about parent_stations, but under the header for OverlappingTripFrequenciesNotice

Reply:
Indeed, thanks! Modifying this.